.github/workflows/component-builder.yml

```
@@ -24,21 +28,24 @@ jobs:
             qemu-system-x86 \
+            qemu-system-aarch64\
             libvirt-daemon-system \
             virtinst cloud-image-utils \
             libguestfs-tools
       - name: Tweak hosted runner to enable 'virt-sysprep'
         # https://bugs.launchpad.net/ubuntu/+source/linux/+bug/759725
         run: sudo chmod 0644 /boot/vmlinuz*
       - name: Fetch base Fedora image
-        working-directory: ./containers/fedora
-        run: wget -q "https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/${{ env.FEDORA_IMAGE }}"
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting

(shellcheck)